shared/elements/gaia_sim_picker/script.js

```diff
@@ -27,8 +27,6 @@ window.GaiaSimPicker = (function(win) {
     LazyLoader.load(['/shared/js/component_utils.js'], function() {
       ComponentUtils.style.call(this, baseurl);
     }.bind(this));
-
-    navigator.mozL10n.ready(this._localizeShadowDom.bind(this));
```

Comment:
Why remove this? The fragment won't get translated when the locale changes without it.